views/error-429.html

```diff
@@ -0,0 +1,95 @@
+
+<!DOCTYPE html>
```

Comment:
Can we add an HTML comment in here to explain why we have this file that is otherwise inaccessible via node? Something like:
```
<!-- Custom static 429 page. See https://bugzilla.mozilla.org/show_bug.cgi?id=1495088 --/>

Reply:
Is it possible to reuse our handlebars templates here and just compile the output to avoid all this duplication and hard coding?
It feels unlikely that we’ll be able to keep this file in sync if it’s hard coded html.

Reply:
Maybe, but I'm not sure it's worth the effort - especially right now. File a follow-up issue?

Reply:
Submitted #491 for tracking+discussion.